### PR TITLE
Fix clearing filters of dot notation relationships

### PR DESCRIPTION
**Pull request by Aaronfischer**
#### What does this pull request do?
Fixes bug where dot notation relationships would be transformed incorrectly when column filter is cleared.

#### What are the relevant issues?
#40 

#### Any background context you want to provide?
Depends upon #39 to fix broken test.

#### Definition of Done:
*Each should be selected to indicate consideration for applicability and/or completion.*

- [x] Does this PR fully satisfy the scope as outlined in the referenced issue(s)?
- [x] Is there appropriate test coverage?
- [x] Is static analysis resulting in a satisfactory score?
- [x] If adding new dependencies, does pip requirements need to be updated?
- [x] Has the change log been updated?

### CODE REVIEW

**Comment by Aaronfischer** [comment]
@morgan Same for this as well, CircleCi fixed within #39.

**Comment by Aaronfischer** [comment]
@morgan Ready for review.

**Comment by ethanpitsch** [comment]
dummy

I've been seeing this last week or two and figured I should mention it at some point. Not sure if it's only happening for me or if everyone is seeing this.

**Comment by Aaronfischer** [comment]
@ethanpitsch Interest catch, looks like this was caused on a recent PR that fixed the ember-try (canary). May be some discrepancy on how to remove component wrappers across versions, fixed canary and it visually broke previous version.
I think I can refactor this pretty simply to do this a different way altogether.

Thanks, I'll create an issue for tracking purposes.

File: tests/acceptance/advanced-test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import moduleForAcceptance from '../helpers/module-for-acceptance';
```
Comment (morgan) [inline tests/acceptance/advanced-test.js]: @Aaronfischer The name of this test is too broad. Let's be more specific. Such a `relationships-test.js`

**Pull request merged**